purchase_advance_payment/views/purchase_view.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <button name="button_draft" position="after">
+                <button
+                    name="%(action_view_account_voucher_wizard)d"
+                    string="Pay purchase advanced"
+                    type="action"
+                    groups="account.group_account_invoice"
+                    invisible="state in ['done', 'cancel'] or not amount_residual"
+                />
+            </button>
+            <notebook position="inside">
+                <page string="Payment advances" groups="account.group_account_invoice">
+                    <field
+                        name="account_payment_ids"
+                        nolabel="1"
+                        colspan="4"
+                        context="{'form_view_ref': 'account.view_account_payment_form','tree_view_ref': 'account.view_account_payment_tree'}"
+                    />
+                </page>
+            </notebook>
+            <field name="tax_totals" position="after">
+                <div class="oe_subtotal_footer_separator oe_inline o_td_label">
+                    <label for="amount_residual" />
+                </div>
+                <field
```

Comment:
The field is not shown as the same format 17.0. Would you adjust this?
version 17.0:
![2025-03-20_11-20](https://github.com/user-attachments/assets/c269f894-bf5e-4fff-a6cd-fc5d6e2f0b8c)

version 18.0: 
![2025-03-20_11-05](https://github.com/user-attachments/assets/bbb3e49d-de77-43ea-8e41-6e55138e3799)